transcoder/output/OutputUtil.py

```diff
@@ -70,6 +75,9 @@ def get_output_manager(output_name: str, output_prefix: str = None, output_file_
         output = DiagnosticOutputManager()
     elif output_name == JsonOutputManager.output_type_identifier():
         output = JsonOutputManager(output_prefix, output_file_path, lazy_create_resources=lazy_create_resources)
+    elif output_name == LengthDelimitedOutputManager.output_type_identifier():
+        # TODO: pass through output endian specification from CLI args
```

Comment:
Should this be a new issue?

Reply:
Yes, and there's also a need for `--input_prefix_length` for `LengthDelimitedFileMessageSource` and `--output_prefix_length` for `LengthDelimitedOutputManager`. I will organize these under a separate, new issue.